Information gain computation logic for Attribution Scope. (#1364)
WICG/attribution-reporting-api: verbose_debugging_reports.md

Additional fields: `limit`

#### `source-scopes-channel-capacity-limit`

> A source is rejected due to the [attribution scope channel-capacity limit][].

Reviewer: ```suggestion
The source was rejected due to the [attribution scope channel-capacity limit][].
```
